ray.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Is this `ray` config file required. Or can we omit it to use default settings (just using environment variables to overwrite)

Reply:
I'm not 100% sure but I think the config file is required. I actually didn't intend on shipping with ray as part of the package so I'm going to update to remove it.